components/director/internal/authenticator/middleware.go

```diff
@@ -49,17 +49,18 @@ func (a *Authenticator) Handler() func(next http.Handler) http.Handler {
 			}
 
 			claims := Claims{}
-
-			token, err := jwt.ParseWithClaims(bearerToken, &claims, a.getKeyFunc())
+			_, err = jwt.ParseWithClaims(bearerToken, &claims, a.getKeyFunc())
 			if err != nil {
```

Comment:
This error check can be combined with line 52:

```go
if _, err = jwt.ParseWithClaims(bearerToken, &claims, a.getKeyFunc()); err != nil {
```

Reply:
Yup, but in this case I think it is more readable like it is, as the `ParseWithClaims` function call is quite complicated.